environments/lcls/__init__.py

```diff
@@ -257,9 +177,9 @@ def get_observables(self, observable_names: List[str]) -> Dict:
 
         if observe_gas:
             intensity_p80, intensity_mean, intensity_median, intensity_std, \
-                loss_p80 = self.get_intensity_n_loss()
+                loss_p80 = fel.observables.get_intensity_n_loss(hxr = self.hxr, points = self.points, loss_pv = self.loss_pv, fel_channel = self.fel_channel, interface = self.interface)
```

Comment:
lets apply black formatting to this file and the observables.py file using the ufmt package https://pypi.org/project/ufmt/